include/rocksdb/wide_columns.h

```diff
@@ -218,4 +220,42 @@ inline bool operator!=(const PinnableWideColumns& lhs,
   return !(lhs == rhs);
 }
 
+// TODO - find me a better name!
```

Comment:
Wanted to put this up for another round of review while I come up with a better name :)